dev-libs/libdispatch/libdispatch-5.5.ebuild

```diff
@@ -46,5 +49,6 @@ src_configure () {
 	export HOST_CXX="$(tc-getBUILD_CXX)"
 	tc-export CC CXX LD AR NM OBJDUMP RANLIB PKG_CONFIG
 
+	local mycmakeargs=( -DBUILD_TESTING=OFF )
```

Comment:
Why?

Reply:
Avoids a warning under musl (easily fixable but eh), and the tests aren't used so I thought it'd make sense to just disable them.

Reply:
Put a comment indicating such then (but maybe they should be enabled?)

Reply:
I'll add a comment indicating that this disables building the testsuite. But why should it be enabled without an src_test?

Reply:
Because we always want to enable running test suites?

and yes, comment, but explain why you're disabling it

Reply:
Sure, I've removed the line and fixed the testing code to not produce a warning. I don't really see the point of building the tests while the package doesn't support testing, but whatever, it builds fast anyway.

Reply:
I don't understand. How does it not support testing? I'm saying it should if tests exist?

Reply:
Oh, like that. Yeah I have no idea why this package has no src_test, I just noticed the option and realized the tests weren't being used anyway.
I guess this is something that should be handled in a separate PR.